modules/templates.js

```diff
@@ -0,0 +1,13 @@
+angular.module('pages').run(['$templateCache', function($templateCache) {
```

Comment:
There are some other projects we have (private ones I think) that have much improved `ngtemplates` configuration code, that does all of this. @grahamgilchrist will know; I think he wrote a standalone file you could bring in